src/sql/migrations.rs

```diff
@@ -649,6 +649,17 @@ CREATE INDEX smtp_messageid ON imap(rfc724_mid);
         )
         .await?;
     }
+    if dbversion < 95 {
+        sql.execute_migration(
+            "CREATE TABLE chats_contacts_new (chat_id INTEGER, contact_id INTEGER, UNIQUE(chat_id, contact_id));\
+            INSERT OR IGNORE INTO chats_contacts_new SELECT * FROM chats_contacts;\
+            DROP TABLE chats_contacts;\
+            ALTER TABLE chats_contacts_new RENAME TO chats_contacts;\
+            CREATE INDEX chats_contacts_index1 ON chats_contacts (chat_id);\
+            CREATE INDEX chats_contacts_index2 ON chats_contacts (contact_id);",
+            95
+        ).await?;
```

Comment:
clever trick, to remain compatible when downgrading for whatever reason. i was first wondering why `OR IGNORE`  can work above :)

nitpicking: i'd use sth. as ` chats_contacts_temp`  instead of `_new` - then it would be even more obvious, that this is not a new table but we just alter the index of the existing one.

Reply:
In fact, I'll rename it to `new_chats_contacts`, so that it matches the migration from #3776